stko/utilities/utilities.py

```diff
@@ -1157,3 +1158,98 @@ def vector_angle(vector1, vector2):
     if term <= -1.:
         return np.pi
     return np.arccos(term)
+
+
+def get_torsion_info_angles(
+    mol: stk.ConstructedMolecule,
```

Comment:
as above with type hints - especially here as it is only done for one argument. I think it requires a lot of changes to start implementing them, so lets remove them for now.

Reply:
Yeah when I was adding types to the docstrings I was just adding them as type hints if it wasn't any harder but makes sense to keep everything consistent for now.